texajson.go

```diff
@@ -204,6 +268,7 @@ func ConvtoSlabPage(ArtiQSA []uint64, SlabNameArray []string, slabSeqArray []str
 	fmt.Println(sp)
 
 	SlabTempNQD = make([]int, len(SlabNameArray))
+	t := 0
```

Comment:
`t` is a very arbitrary name. Maybe use the word `TransactionCounter`?

Reply:
Done.